client/my-sites/media/main.jsx

```diff
@@ -358,6 +359,20 @@ class Media extends Component {
 				{ mediaId && site && site.ID && <QueryMedia siteId={ site.ID } mediaId={ mediaId } /> }
 				<PageViewTracker path={ this.getAnalyticsPath() } title="Media" />
 				<DocumentHead title={ translate( 'Media' ) } />
+				{ this.props.selectedSite.is_private && this.props.selectedSite.is_wpcom_atomic && (
+					<Notice
```

Comment:
Should the `<Notice>` appear below the heading? What do we do elsewhere?

<img width="1631" alt="image" src="https://user-images.githubusercontent.com/36432/218501777-a76cfb9d-b7d7-4a11-b1d6-c3831b128e4d.png">

Reply:
Thanks! I found other Notices on the plans and domain pages that were below the headers, so I moved this <Notice> below the <FormattedHeader> component.